.github/workflows/ci-additional.yaml

```diff
@@ -134,18 +134,16 @@ jobs:
           name: codecov-umbrella
           fail_ci_if_error: false
 
-  mypy38:
-    name: Mypy 3.8
+  mypy39:
```

Comment:
Is this needed any more? I thought it was a one-off for 3.8.

cc @Illviljan @headtr1ck

Reply:
I think we should continue checking the minimum python. I think it has proven it's importance since we missed that you had to use `typing.Tuple`, `typing.List` etc. until 3.9.